docs/source/get_started/install.md

```diff
@@ -84,36 +45,41 @@ To exit the environment:
 ```bash
 deactivate
 ```
-</details>
 
-<details>
-<summary><b>Click to expand instructions for <code>pipenv</code></b></summary>
 
-Install `pipenv` as follows:
+### How to create a new virtual environment without using `venv`
```

Comment:
"without `venv`" implies that we're covering more options. I would just rename this to

```suggestion
### How to create a new virtual environment using `conda`
```